content/learn.md

```diff
@@ -341,8 +393,41 @@ dasharo_faq:
 
     - title: What is the payment policy and who pays for Dasharo?
       content: >-
-        Typically, firmware solutions are sold through royalty payment or subscription-based models. The first option relies on the volume of offered products, what effectively eliminates industries with low volume but high-value platforms e.g. medical robots. Subscription-based models, although better, are still tied to firmware solution providers and similarly lead to all the scope of negative effects of vendor lock-in. Dasharo’s pricing model does not rely on volume and since most components are open-source, you can always change vendor and re-own your firmware code. Dasharo can be purchased by OEM/ODM providers, community crowdfunding, individuals, companies that are looking for dedicated/confidential firmware solutions. To get more dedicated payment information, fill [Get a quote](/#get-a-quote).
-      
+        Typically, firmware solutions are sold through royalty payment or subscription-based models. The first option relies on the volume of offered products, what effectively eliminates industries with low volume but high-value platforms e.g. medical robots. Subscription-based models, although better, are still tied to firmware solution providers and similarly lead to all the scope of negative effects of vendor lock-in. 
+
+    - title: What services include in the price plan?
+      content: >-
+        First, there is no license fee. By choosing the support package, we start cooperation for a longer period of at least half a year. The plan is presented at the beginning of each month. The payment depends on the senior developer's time worked. exceeding the working time inconsistent with the assumed plan is reported to the client. We also assume time for management and meetings with the client, which is necessary to run the project.
+
+    - title: What pricing model do you offer?
+      content: >-
+        Dasharo’s pricing model does not rely on volume and since most components are open-source, you can always change vendor and re-own your firmware code. Dasharo can be purchased by OEM/ODM providers, community crowdfunding, individuals, companies that are looking for dedicated/confidential firmware solutions. To get more dedicated payment information, fill [Get a quote](/#get-a-quote).
+
+    - title: Can I introduce UEFI firmware into my existing platform/estate?
+      content: >-
+        [Find out here](//localhost:1313/pages/learn/#q-general-9)
+
+    - title: Can I implement coreboot (as boot fw) + iPXE (as payload) to my device?
+      content: >-
+        [Find out here](//localhost:1313/pages/learn/#q-general-9)
+
+    - title: Can I port Core boot to use in my production environment?
+      content: >-
+        [Find out here](//localhost:1313/pages/learn/#q-general-9)
```

Comment:
@MateuszKochner that's not a correct link - see the examples above, like in the `Get a quote`.

Reply:
corrected